html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML forms</title>
+  </head>
+   <body>
+     <form>
+    <h1>Coding Forms!</h1>
+     <br />
+     <div>Email:</div>
+      <label for= "Email">
+        <input id= "Email" type= "email" name= "Email" size= "30" maxlength= "100">
+      </label>
```

Comment:
Like this 
![image](https://user-images.githubusercontent.com/52118958/99885815-6a433e00-2bec-11eb-96a0-48ded464b30a.png)